pkg/k8s/network_policy_v1beta_test.go

```diff
@@ -107,15 +107,24 @@ func (s *K8sSuite) TestParseNetworkPolicyDeprecated(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(result, comparator.DeepEquals, &policy.L4Policy{
 		Ingress: policy.L4PolicyMap{
-			"80/TCP": policy.L4Filter{
-				Port: 80, Protocol: api.ProtoTCP, U8Proto: 6,
-				FromEndpoints:  []api.EndpointSelector{epSelector},
-				L7Parser:       "",
-				L7RedirectPort: 0, L7RulesPerEp: policy.L7DataMap{},
-				Ingress: true,
+			Filters: map[string]policy.L4Filter{
+				"80/TCP": {
+					Port: 80, Protocol: api.ProtoTCP, U8Proto: 6,
+					FromEndpoints:  []api.EndpointSelector{epSelector},
+					L7Parser:       "",
+					L7RedirectPort: 0, L7RulesPerEp: policy.L7DataMap{},
+					Ingress: true,
+				},
+			},
+			SourceRuleLabels: labels.Labels{
+				"unspec:io.cilium.k8s-policy-name":              labels.ParseLabel("unspec:io.cilium.k8s-policy-name"),
```

Comment:
This is not respecting the documentation of the Labels struct:

https://github.com/cilium/cilium/blob/5fc5b37a2b9d31da0bef6ea61eaac63f7fdf24ff/pkg/labels/labels.go#L151-L152


The map's key should be the same as the labels' key.